api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the operator.kyma-project.io v1alpha1 API group
+// +kubebuilder:object:generate=true
+// +groupName=operator.kyma-project.io
+package v1alpha1
```

Comment:
The sibling PR from Xin also doesn't have this exception. If we need it, it can be added with the controller. for the scope of this PR, it doesn't seem essential

Reply:
please resolve conversation if you agree